spring-auto-restdocs-core/src/test/java/capital/scalable/restdocs/util/FormatUtilTest.java

```diff
@@ -44,6 +44,8 @@ public void addDot() {
         assertThat(FormatUtil.addDot("."), is("."));
         assertThat(FormatUtil.addDot("text."), is("text."));
         assertThat(FormatUtil.addDot("text"), is("text."));
+        assertThat(FormatUtil.addDot("text with html:\n<ul>\n<li>first</li>\n<li>second</li>\n</ul>"),
+                is("text with html:\n<ul>\n<li>first</li>\n<li>second</li>\n</ul>"));
```

Comment:
this test only tests `</ul>` part of the regex. it would be nice to add more tests for the other text-ending characters you described in PR to ensure that it's correct.

Reply:
it's just a test, but good point, it would be nice to add another tests with different line separators to ensure correct multiline regex behavior.

Reply:
I agree with that and added more test cases. :+1: